.github/tests/pre-install.sh

```diff
@@ -47,3 +47,8 @@ kubectl wait --namespace ingress-nginx --for=condition=ready --timeout 60s pod -
   --namespace postgresql \
   --values "${DEPS}/postgresql.yaml" \
   --wait
+
+# kyverno
+"${helm_install[@]}" kyverno kyverno --version "$VERSION_KYVERNO" --repo "$HELM_REPO_KYVERNO" \
```

Comment:
Let's not slip in a policy engine as an out-of-scope "extra" on a commit intended to add a CSI driver.